lib/cli/ui/prompt.rb

```diff
@@ -48,35 +55,47 @@ class << self
         # Free form question when the answer can be empty
         #   CLI::UI::Prompt.ask('What is your opinion on this question?', allow_empty: true)
         #
-        # Question with answers
-        #   CLI::UI::Prompt.ask('What kind of project is this?', options: %w(rails go ruby python))
+        # Interactive (multiple choice) question
+        #   CLI::UI::Prompt.ask_interactive('What kind of project is this?', options: %w(rails go ruby python))
         #
+        # Interactive (multiple choice) question with defined handlers
+        #   CLI::UI::Prompt.ask_interactive('What kind of project is this?', options: %w(rails go ruby python)) do |handler|
```

Comment:
`ask` here too

Reply:
We should split it up to:
```ruby
CLI::UI::Prompt.ask('What kind of project is this?', options: %w(rails go ruby python))
```

and 

```ruby
CLI::UI::Prompt.ask('What kind of project is this?') do |handler|
  handler.options('A') { }
end
```

Also `puts selection` in the blocks below returns nil, not selection ;)

Reply:
Ah, I meant just "outputs selection", but i see the `=>` notation is confusing.